ow_lander/urdf/lander.xacro

```diff
@@ -844,7 +844,7 @@
     <axis
       xyz="0 0 -1" />
     <dynamics damping="50.0" />
-    <limit effort="80.0" velocity="0.2" />
+    <limit effort="80.0" velocity="0.2" lower="-3.14" upper="3.14" />

```

Comment:
There's a tiny gap in the rotation range here. I suspect a real pan joint would be engineered to rotate a little more than 2*pi to avoid getting stuck in some state where it has to wrap around over and over again. Maybe -3.5 to 3.5 or something like that.

Reply:
Makes sense

Reply:
increase from 3.14 to 3.50 +/-